examples/c/sockfilter.c

```diff
@@ -56,10 +56,17 @@ static int libbpf_print_fn(enum libbpf_print_level level, const char *format, va
 	return vfprintf(stderr, format, args);
 }
 
+static inline void ltoa(uint32_t addr, char *dst)
+{
+	snprintf(dst, 16, "%d.%d.%d.%d", (addr >> 24) & 0xFF, (addr >> 16) & 0xFF,
```

Comment:
should this be %u?

Reply:
The range of each number is between 0 - 255, we can use `%u` to make sure no negative number will appear.